### DISCUSSION TITLE
Service Worker Error

### DISCUSSION THREAD

**Discussion by vsjha18** [coder/code-server#2173] [Q&A]
code-server 3.5.0 [standalone version]

Each time I open any extension with `webview` I see the following error
![image](https://user-images.githubusercontent.com/1805276/95464057-275c3c80-0997-11eb-9ed0-6368cb5a5490.png)

It can be seen both in Chrome and Safari. Is there any way to mute it. Because functionally everything seems fine.

**Comment by code-asher** [comment]
There's no way to mute the message but if you use a secure context
(https) then service workers will be enabled and the message won't
appear.

I'm not certain what exactly doesn't work without service workers but I
imagine extensions that make use of the webview API will have trouble.

**Reply by vsjha18** [reply]
Got your point. I will make it HTTPS. Feature works exactly fine, only the pop-up message comes.
